beps/bep_0003.rst

```diff
@@ -87,70 +92,115 @@ announce
 
 info
   This maps to a dictionary, with keys described below.
+  
+``piece layer``
```

Comment:
Just to clarify once more: It's just *one* level of the tree *for each file*. Not the levels above - they can be derived if needed - or the levels below - they would take up too much space.